tests/functional/get_v1_account/test_get_v1_account.py

```diff
@@ -1,50 +1,51 @@
 from datetime import datetime
-
 from hamcrest import (
     assert_that,
     all_of,
     has_property,
-    instance_of,
     has_properties,
     contains_inanyorder,
     starts_with,
     equal_to,
     has_item,
     )
-
+from checkers.http_checkers import check_status_code_http
 from dm_api_account.models.Response.user_envelope import UserRole
```

Comment:
```suggestion
from dm_api_account.models.response.user_envelope import UserRole
```
называй модули по питонячь

Reply:
исправил , там и рексест и респонс и во всех зависимостях )))